CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -363,6 +363,13 @@ CF_PRIVATE Boolean __CFProcessIsRestricted();
 CF_EXPORT void * __CFConstantStringClassReferencePtr;
 CF_EXPORT void *__CFConstantStringClassReference[];
 
+#if __CF_BIG_ENDIAN__
+#define CFINFO {0x00, 0x00, 0x07, 0xc8}
+#elif __CF_LITTLE_ENDIAN__
+#define CFINFO {0xc8, 0x07, 0x00, 0x00}
+#endif
```

Comment:
I also have a PR https://github.com/apple/swift-corelibs-foundation/pull/392 which should address the CFInfo field for endianness.  It actually collapses the various definitions into a single one as well.